apparmor.d/groups/lxqt/lxqt-config-powermanagement

```diff
@@ -0,0 +1,41 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-config-powermanagement
+profile lxqt-config-powermanagement @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/bus-system>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+
+  @{exec_path} mr,
+
+  /etc/machine-id r,
+
+  owner @{user_config_dirs}/lxqt/#@{int} rw,
+  owner @{user_config_dirs}/lxqt/lxqt-powermanagement.conf.lock rwk,
+  owner @{user_config_dirs}/lxqt/lxqt-powermanagement.conf.@{rand6} rw,
+  owner @{user_config_dirs}/lxqt/lxqt-powermanagement.conf.@{rand6} l -> @{user_config_dirs}/lxqt/#@{int},
+
+  owner /tmp/@{int} r,
+
+  @{sys}/class/backlight/ r,
+  @{sys}/devices/@{pci}/**/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/* rw,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/ r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/max_brightness r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/bl_power r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/actual_brightness  r,
+  @{sys}/devices/@{pci}/**/**/drm/card@{int}/card@{int}-eDP-1/amdgpu_bl@{int}/* r,
```

Comment:
That cannot be let like this:
- This is intel/amd specific
- This is specific to a display port output.
- `0000:00:02.0` is part of the pci address. This address often refers to the first GPU connected.

Have a look at the profile `xfpm-power-backlight-helper` and copy all sys rules from it. This is what you need here.